hypothesis/lms
# Remove UI prototypes for already implemented features

This PR removes the UI playground pages for features that are already implemented.

They served a purpose, but now it's code we don't want to maintain anymore, as the features they reference to, can now evolve and differ from what was originally prototyped.

### Testing steps

If you check out this branch and go to http://localhost:8001/ui-playground/, it should display the same as https://patterns.hypothes.is/, with no additional pages.
